### PR TITLE
UE-263: Style and set up a hard-coded fat footer with links

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
VPUE fat footer below the local footer and ABOVE global footer. 
Tightening up the webpack output
Removing commented out old code.

# Review By (Date)
- Soon

# Urgency
- High

# Steps to Test

1. Pull in the changes.
2. run `npm run publish && lando drush cr`
3. Verify build works and the footer looks as wanted.

# Affected Projects or Products
- VPUE Subtheme
- Footer

# Associated Issues and/or People
- [UE-263](https://stanfordits.atlassian.net/browse/UE-263): Style and set up a hard-coded fat footer with links
- @jdwjdwjdw 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: templates/components/fat-footer/vpue-fat-footer.html.twig [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+{% import "@jumpstart_ui/macros/field-links-to-social-list.twig" as sociallist %}
+
+
+<div class="vpue-fat-footer su-local-footer">
+  <div class="vpue-gradient-divider"></div>
+  <div class="su-local-footer__header vpue-fat-footer__header">
+    <div class="vpue-fat-footer__logo-container">
+      <img src="{{ base_path ~ directory }}/dist/assets/svg/vpue-logo.svg" alt="Stanford VPUE" class="vpue-fat-footer__logo" />
+    </div>
+    {# <div class="su-lockup su-lockup--option-k">
```
Comment (jenbreese) [inline templates/components/fat-footer/vpue-fat-footer.html.twig:12]: If it is commented out, can we remove it?

**Comment by jenbreese** [comment]
hi @buttonwillowsix , I only assigned this to you because Github doesn't find Jacob as a reviewer? I made some changes to show him for the footer.

**Comment by pookmish** [comment]
FYI: You can't assign Jacob because he is the PR creator. The creator can't "review" or be assigned to their own PR.

**Comment by jdwjdwjdw** [comment]
@jenbreese Thanks for the review/cleaning things up! Looking good on my local, should be ready to merge.

**Pull request merged**